--- a/config.mk	2022-10-16 22:35:09.405000406 +0800
+++ b/config.mk	2022-10-16 22:36:37.735000411 +0800
@@ -4,17 +4,17 @@
 FRIDA := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 
 # Features ordered by binary footprint, from largest to smallest
-FRIDA_V8 ?= auto
+FRIDA_V8 ?= disabled
 FRIDA_CONNECTIVITY ?= enabled
-FRIDA_DATABASE ?= enabled
-FRIDA_JAVA_BRIDGE ?= auto
-FRIDA_OBJC_BRIDGE ?= auto
-FRIDA_SWIFT_BRIDGE ?= auto
+FRIDA_DATABASE ?= disabled
+FRIDA_JAVA_BRIDGE ?= enabled
+FRIDA_OBJC_BRIDGE ?= enabled
+FRIDA_SWIFT_BRIDGE ?= enabled
 
 FRIDA_AGENT_EMULATED ?= yes
 
 # Include jailbreak-specific integrations
-FRIDA_JAILBREAK ?= auto
+FRIDA_JAILBREAK ?= disabled
 
 FRIDA_ASAN ?= no
 
